.vscode/settings.json

```diff
@@ -1,8 +0,0 @@
-{
-    // enable globally (here: format on save)
-"editor.formatOnSave": true
-// enable per-language (here: Prettier as formatter)
-"[javascript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-}
-}
```

Comment:
Why did you delete this? Some useful VSCode settings can probably be in the repo.

Reply:
Hm.. I think I thought that it's not the best idea to have some vs code adjustments in project itself. But if you think  it's alright, let's keep it. 

Is there a reason why you changed button color to primary? Because now for example claim, header, search has everywhere white button, where it should be yellow

Reply:
I'm thinking we should use the yellow more sparingly, to draw the user's attention. Something like the example claims can have the white button.

Reply:
So decided to leave colors as it is, I mean with your changes. If I understood correctly according to new design all components that I've mentioned should change as well with it. And maybe it would be easier to agree with colors when we will have it in the big picture.